src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/ListPackage/ListPackageCommandRunner.cs

```diff
@@ -25,6 +25,18 @@ public class ListPackageCommandRunner : IListPackageCommandRunner
         private const string ProjectAssetsFile = "ProjectAssetsFile";
         private const string ProjectName = "MSBuildProjectName";
 
+        private static readonly Func<InstalledPackageReference, bool> TopLevelPackagesFilterForOutdated =
```

Comment:
If you make these internal, you can unit tests for these funcs :) 
Not sure if it's necessary, you might have enough coverage otherwise, so take this with a grain of salt.

Reply:
Done https://github.com/NuGet/NuGet.Client/pull/2959/commits/f1c6f4441d86271abcc62356cd2e3a2d39effa97